### DISCUSSION TITLE
v0.36.0-20220102.161018

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#890] [General]
* Neovim 0.6.1 😀
* GH-876: Forward `Cmd-.` to Neovim. Thank you @imkerberos!
* GH-881: Make the `ESC` key cancel "Not yet saved"-dialogs. Thank you @jasonm!
* GH-874: Make Chinese input possible (and simplify input method handling in general), thank you very much, @SolaWing!
* GH-805 (GH-415): Vertically center-align text. Thank you @apaleslimghost!
* Improve menu item key equivalent handling.
* Dependencies updates:
    - sindresorhus/github-markdown-css@5.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.36.0'>v0.36.0-20220102.161018</a>.</em>

**Comment by jasonm** [comment]
@qvacua Thanks for the release! Happy new year 😄

**Reply by qvacua** [reply]
Happy new year to you, too!
